rollbar-yesod/rollbar-yesod.cabal

```diff
@@ -51,6 +51,7 @@ library
     , unliftio >=0.2 && <1
     , wai >=3.2 && <4
     , yesod-core >=1.6 && <2
+    , bytestring >= 0.10
```

Comment:
I don't think you are making use of the dependency in the library code, so this could be removed.